setup.bash

```diff
@@ -42,12 +42,18 @@ CUDA_MAJOR=$(echo "$CUDA_VERSION" | cut -d. -f1)
 CUDA_MINOR=$(echo "$CUDA_VERSION" | cut -d. -f2)
 if [ "$CUDA_MAJOR" -lt $CUDA_MIN_VER_MAJOR ] ||
     { [ "$CUDA_MAJOR" -eq $CUDA_MIN_VER_MAJOR ] && [ "$CUDA_MINOR" -lt $CUDA_MIN_VER_MINOR ]; }; then
-    echo "CUDA version not supported! Get $CUDA_MIN_VER_MAJOR.$CUDA_MIN_VER_MINOR+" && exit 1;
+    echo "CUDA missing or CUDA version not supported! Get CUDA $CUDA_MIN_VER_MAJOR.$CUDA_MIN_VER_MINOR+";
+    echo "Remember to extend environment variables:
+    - PATH with CUDA bin directory
+    - LD_LIBRARY_PATH with CUDA lib directory";
```

Comment:
This is not needed, on my machine it is empty and it works. I think `nvcc` finds libs relatively to its location.